aztec/src/main/kotlin/org/wordpress/aztec/Aztec.kt

```diff
@@ -23,17 +23,29 @@ open class Aztec private constructor(val visualEditor: AztecText, val sourceEdit
     private var onAudioTappedListener: AztecText.OnAudioTappedListener? = null
     private var onMediaDeletedListener: AztecText.OnMediaDeletedListener? = null
     private var plugins: ArrayList<IAztecPlugin> = visualEditor.plugins
+    var sourceEditor: SourceViewEditText? = null
 
     init {
         initHistory()
         initToolbar()
     }
-    
+
     constructor(activity: Activity, @IdRes aztecTextId: Int,
```

Comment:
All constructors should be private as we only want to use the factory to create an instance.